R/available_forecasts.R

```diff
@@ -70,23 +70,7 @@ available_forecasts <- function(data,
   out <- merge(out, out_empty, by = by, all.y = TRUE)
   out[, count := nafill(count, fill = 0)]
 
-  class(out) <- c("scoringutils_available_forecasts", class(out))
+  class(out) <- c("forecast_counts", class(out))
```

Comment:
Very good point, we should definitely avoid that. 
`scoringutils_counts`? `counts`? `prediction_counts`? 

I think I like the third one best.

Reply:
I changed it to `prediction_counts` for now. Maybe it would be good to devise a more systematic naming convention. 
At the moment we have our 4 main classes for the forecast types, but then we're also planning to introduce all kinds of one-hit-wonder classes for plotting. Maybe prefixing them with `scu_` or something like that would be a good idea?